internal/thelma/app/logging/logging.go

```diff
@@ -40,6 +40,10 @@ type logConfig struct {
 	}
 }
 
+func init() {
+	Bootstrap()
```

Comment:
just curious, was this not required before?

Reply:
great question. It's called explicitly in an init function that lives in the `app` package: https://github.com/broadinstitute/thelma/blob/main/internal/thelma/app/app.go#L23. This works fine for when we run Thelma as a command, and during bulk integration tests runs (because a subset of the tests do cause the `app` package to be loaded).

But lots of tests, don't depend on the `app` package, but DO depend on logging. This includes tests in the `vault` package.

Adding this to an init function in the logging package allowed me to confirm masking was working in when I manually ran individual unit tests in just the`vault` package:
```
3:11PM DBG automatically masked 1 fields in Vault response
3:11PM DBG GET /v1/secret/foo/bar
3:11PM DBG returning secret foo/bar: map[abc:******]
3:11PM DBG 200 response={"data":{"abc":"******"},"lease_duration":0,"lease_id":"","renewable":false,"request_id":"","warnings":null}
3:11PM DBG automatically masked 1 fields in Vault response
```

Reply:
I've removed this from the app package's `init` function. It's harmless to run it both places (it's idempotent), but crufty.